src/components/Filter/EnumFilter.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useMemo, useState } from 'react';
-import { useTranslation } from 'src/internal/i18n';
+import { useTranslation } from 'src/utils/i18n';
```

Comment:
GH suggests auto-fixing this linter warning (import order is not sorted)

Reply:
Nice, I think it's fixed now, waiting for the tests